### ISSUE TITLE
screen_state unrealistically high minSdkVersion

### ISSUE THREAD

**Issue by sc00n** [cph-cachet/flutter-plugins#793]
screen_state, used by the packages for mobile sensing from CARP which I am using, has currently a minSdkVersion of 31 as set in the package build.gradle:

```
    defaultConfig {
        minSdkVersion 31
    }
```

In principle I should therefore also increase my minSdkVersion to 31. This means however that my app will not be available for about 70% of Android users. 

By adding

`uses-sdk tools:overrideLibrary="dk.cachet.screen_state"/`

in my manifest I can circumvent it, but I don't know if it will break something with regards to mobile sensing. Without screen_state I can bring the whole app down to minSdkVersion of 23, which will probably work for 99% if Android users. 

To summarize, I have two questions:

1. Would it be possible to decrease the minSdkVersion for screen_state
2. What will break if I ignore the minSdkVersion

Thanks!
